lambdas/preview/requirements.txt

```diff
@@ -0,0 +1,47 @@
+# TODO: add "../shared/" or "t4-lambda-shared" in a way that's not totally broken
+atomicwrites==1.3.0
+attrs==18.2.0
+backcall==0.1.0
+bleach==3.1.0
+certifi==2018.11.29
+chardet==3.0.4
+codecov==2.0.15
+coverage==4.5.2
+decorator==4.3.2
+entrypoints==0.3
+idna==2.7
+ipython==7.2.0
+ipython-genutils==0.2.0
+jedi==0.13.2
+Jinja2==2.10
+jsonschema==2.6.0
+jupyter-core==4.4.0
+MarkupSafe==1.1.0
+mistune==0.8.4
+more-itertools==6.0.0
+nbconvert==5.3.1
+nbformat==4.4.0
+numpy==1.16.1
+pandas==0.24.1
+pandocfilters==1.4.2
+parso==0.3.4
+pexpect==4.6.0
+pickleshare==0.7.5
+pluggy==0.8.1
+prompt-toolkit==2.0.8
+ptyprocess==0.6.0
+py==1.7.0
+pyarrow==0.12.0
+Pygments==2.3.1
+pytest==4.2.1
```

Comment:
the [tests] extras are also in requirements.txt (pinned to a specific verision) -- but why would we send the tests requirements to lambda?

Reply:
Oh, good point